modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -40,230 +44,209 @@
 public class ResidentIDPConfigsTestCase extends ISIntegrationTest {
 
     private IdentityProviderMgtServiceClient idpMgtServiceClient;
-    private AuthenticatorClient loginManger;
 
-    @BeforeClass(alwaysRun = true)
-    public void testInit() throws Exception {
+    private final String username;
+    private final String userPassword;
+    private final String activeTenant;
+    private Map<String, String> fedAuthConfigMap;
+    private Map<String, String> provisioningConfigMap;
+    private final String SUPER_TENANT = "carbon.super";
+    private final String TENANT_DOMAIN = "wso2.com";
+
+    @DataProvider(name = "configProvider")
+    public static Object[][] configProvider() {
+        return new Object[][]{
+                {TestUserMode.SUPER_TENANT_ADMIN},
+                {TestUserMode.TENANT_ADMIN}
+        };
+    }
 
-        super.init();
-        ConfigurationContext configContext = ConfigurationContextFactory.createConfigurationContextFromFileSystem(null, null);
-        idpMgtServiceClient = new IdentityProviderMgtServiceClient(sessionCookie, backendURL, configContext);
+    @Factory(dataProvider = "configProvider")
+    public ResidentIDPConfigsTestCase(TestUserMode userMode) throws Exception {
+
+        super.init(userMode);
+        AutomationContext context = new AutomationContext("IDENTITY", userMode);
+        this.username = context.getContextTenant().getTenantAdmin().getUserName();
+        this.userPassword = context.getContextTenant().getTenantAdmin().getPassword();
+        this.activeTenant = context.getContextTenant().getDomain();
     }
 
-    @Test(groups = "wso2.is", description = "Test resident IdP config URLs in super tenant mode")
-    public void testResidentIdPConfigs() throws Exception {
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
 
-        log.info("Retrieving resident identity provide");
```

Comment:
Isn't it good to have a log to track the status of the tests execution?

Reply:
Fixed https://github.com/wso2/product-is/pull/9463/commits/40f51782b1b30ff3722701becd32b4020dcf84b8